packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -72,10 +28,9 @@ Tabs
         isSelected (boolean, default = false) // Select first tab by default?
         hasIcon (boolean, default = false)
         hasCount (boolean, default = false)
-    Tab (2+ required)
+    Hds::Tabs.Tab (2+ required)
     ...
     ScrollNavigation (displays if more tabs than fit container)
-  TabPanel
-  TabPabel
+  Hds::Tabs.TabPanel (2+ required, same # as Tab components)
```

Comment:
Maybe, since now `tab`s and `panel`s are all yielded a siblings inside the `Tabs`, I would us just `Panel` instead of `TabPanel` (so `Hds::Tabs::Panel`). Unless we want to maintain the exact naming as its "role".

Reply:
Updated naming. Refactored class names also to use BEM style